addons/mail_bot/models/mail_bot.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, _
+import random
+
+
+class MailBot(models.AbstractModel):
+    _name = 'mail.bot'
+
+    def _logic(self, record, values, command=None):
+        odoobot_id = self.env['ir.model.data'].xmlid_to_res_id("mail_bot.partner_odoobot")
+        if len(record) != 1 or values.get("author_id") == odoobot_id:
+            return
+        if self._pinging_odoobot(values) or self._private_channel_with_odoobot(record):
+            body = values.get("body", "").replace(u'\xa0', u' ').strip().lower().strip(".?!")
+            answer = self._answer(record, body, values, command)
+            if answer:
+                message_type = values.get('message_type', 'comment')
+                subtype_id = values.get('subtype_id', self.env['ir.model.data'].xmlid_to_res_id('mail.mt_comment'))
+                record.with_context({"mail_create_nosubscribe": True}).message_post(body=answer, author_id=odoobot_id, message_type=message_type, subtype_id=subtype_id)
+
+    def _answer(self, record, body, values, command=False):

```

Comment:
Not my idea
We will keep it like that :D 
More than that, it is a sanitized body, so that we avoid to make the same operation each time. 
.replace(u'\xa0', u' ').strip().lower().strip(".?!")